docs/Developers/Executing_Tests.rst

```diff
@@ -56,8 +56,12 @@ To run our tests, you can also use ``python3 setup.py test``
 
 .. note::
 
-    You will not get a test coverage of 100% - the coverage on the
-    website is merged for several python versions.
+    You will not get a test coverage of 100% locally! - the coverage is
+    measured across builds against several python versions. The coverage
+    published on `codecov.io <https://codecov.io/gh/coala/>`__ (GitHub
+    Projects) and `codecov.io <https://codecov.io/gl/coala/>`__ (GitLab
+    Projects) are actually merged for several python versions. Thus, your
+    test coverage can 'pass' even without reaching 100% .
```

Comment:
merged -> merged results & even without -> ~~even~~ without

Reply:
Also results are merged from different OS.  Appveyor results provide coverage of Windows specific lines, and Travis/Circle provide coverage of Unix specific lines.

Reply:
Cool. Let me make changes and make it more clear.